app/build.gradle

```diff
@@ -11,8 +11,8 @@ final ERROR_MESSAGE = "You have not defined an app key for the Onfido api."
 
 def apiKey = getValueFromFileOrEnv(APP_TOKEN_ENV_VAR_NAME, APP_TOKEN_FILE, APP_TOKEN_KEY, ERROR_MESSAGE)
 
-def sdkVersion = '4.0.0'
-def versionCodeNumber = 400
```

Comment:
why are we making the version number so big? With this approach you cannot distinguish `4.1.0` from `41.0.0`. Also `versionName` is now a big redundant, it will show up as `410-sdk:4.1.0`